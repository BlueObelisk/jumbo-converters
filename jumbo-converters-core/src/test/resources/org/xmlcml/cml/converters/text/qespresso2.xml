<?xml version="1.0" encoding="UTF-8"?>
<module templateRef="qespresso" xmlns="http://www.xml-cml.org/schema">
 <module lineCount="1" templateRef="cartax">
  <l xmlns="">Cartesian axes</l>
 </module>
 <module lineCount="2" templateRef="site">
  <l xmlns="">site n. atom positions (a_0 units)</l>
  <l xmlns="">1 Cu tau( 1) = ( 0.0000000 0.0000000 0.0000000 )</l>
 </module>
 <module lineCount="31" templateRef="kpoints">
  <l xmlns="">number of k points= 29 gaussian broad. (Ry)= 0.0200 ngauss = 0</l>
  <l xmlns="">cart. coord. in units 2pi/a_0</l>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 1) = ( 0.0000000 0.0000000 0.0000000), wk = 0.0039062</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 2) = ( -0.1250000 0.1250000 -0.1250000), wk = 0.0312500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 3) = ( -0.2500000 0.2500000 -0.2500000), wk = 0.0312500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 4) = ( -0.3750000 0.3750000 -0.3750000), wk = 0.0312500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 5) = ( 0.5000000 -0.5000000 0.5000000), wk = 0.0156250</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 6) = ( 0.0000000 0.2500000 0.0000000), wk = 0.0234375</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 7) = ( -0.1250000 0.3750000 -0.1250000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 8) = ( -0.2500000 0.5000000 -0.2500000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 9) = ( 0.6250000 -0.3750000 0.6250000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 10) = ( 0.5000000 -0.2500000 0.5000000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 11) = ( 0.3750000 -0.1250000 0.3750000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 12) = ( 0.2500000 0.0000000 0.2500000), wk = 0.0468750</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 13) = ( 0.0000000 0.5000000 0.0000000), wk = 0.0234375</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 14) = ( -0.1250000 0.6250000 -0.1250000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 15) = ( 0.7500000 -0.2500000 0.7500000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 16) = ( 0.6250000 -0.1250000 0.6250000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 17) = ( 0.5000000 0.0000000 0.5000000), wk = 0.0468750</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 18) = ( 0.0000000 0.7500000 0.0000000), wk = 0.0234375</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 19) = ( 0.8750000 -0.1250000 0.8750000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 20) = ( 0.7500000 0.0000000 0.7500000), wk = 0.0468750</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 21) = ( 0.0000000 -1.0000000 0.0000000), wk = 0.0117188</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 22) = ( -0.2500000 0.5000000 0.0000000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 23) = ( 0.6250000 -0.3750000 0.8750000), wk = 0.1875000</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 24) = ( 0.5000000 -0.2500000 0.7500000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 25) = ( 0.7500000 -0.2500000 1.0000000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 26) = ( 0.6250000 -0.1250000 0.8750000), wk = 0.1875000</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 27) = ( 0.5000000 0.0000000 0.7500000), wk = 0.0937500</l>
  </module>
  <module lineCount="1" templateRef="kp">
   <l xmlns="">k( 28) = ( -0.2500000 -1.0000000 0.0000000), wk = 0.0468750</l>
  </module>
  <l xmlns="">k( 29) = ( -0.5000000 -1.0000000 0.0000000), wk = 0.0234375</l>
 </module>
 <l xmlns=""/>
 <l xmlns="">G cutoff = 344.1848 ( 6735 G-vectors) FFT grid: ( 27, 27, 27)</l>
 <l xmlns="">G cutoff = 114.7283 ( 1243 G-vectors) smooth grid: ( 15, 15, 15)</l>
</module>
