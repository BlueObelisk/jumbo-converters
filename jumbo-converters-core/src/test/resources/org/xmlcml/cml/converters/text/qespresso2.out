


   Cartesian axes

     site n.     atom                  positions (a_0 units)
         1           Cu  tau(  1) = (   0.0000000   0.0000000   0.0000000  )

     number of k points=   29  gaussian broad. (Ry)=  0.0200     ngauss =   0
                       cart. coord. in units 2pi/a_0
        k(    1) = (   0.0000000   0.0000000   0.0000000), wk =   0.0039062
        k(    2) = (  -0.1250000   0.1250000  -0.1250000), wk =   0.0312500
        k(    3) = (  -0.2500000   0.2500000  -0.2500000), wk =   0.0312500
        k(    4) = (  -0.3750000   0.3750000  -0.3750000), wk =   0.0312500
        k(    5) = (   0.5000000  -0.5000000   0.5000000), wk =   0.0156250
        k(    6) = (   0.0000000   0.2500000   0.0000000), wk =   0.0234375
        k(    7) = (  -0.1250000   0.3750000  -0.1250000), wk =   0.0937500
        k(    8) = (  -0.2500000   0.5000000  -0.2500000), wk =   0.0937500
        k(    9) = (   0.6250000  -0.3750000   0.6250000), wk =   0.0937500
        k(   10) = (   0.5000000  -0.2500000   0.5000000), wk =   0.0937500
        k(   11) = (   0.3750000  -0.1250000   0.3750000), wk =   0.0937500
        k(   12) = (   0.2500000   0.0000000   0.2500000), wk =   0.0468750
        k(   13) = (   0.0000000   0.5000000   0.0000000), wk =   0.0234375
        k(   14) = (  -0.1250000   0.6250000  -0.1250000), wk =   0.0937500
        k(   15) = (   0.7500000  -0.2500000   0.7500000), wk =   0.0937500
        k(   16) = (   0.6250000  -0.1250000   0.6250000), wk =   0.0937500
        k(   17) = (   0.5000000   0.0000000   0.5000000), wk =   0.0468750
        k(   18) = (   0.0000000   0.7500000   0.0000000), wk =   0.0234375
        k(   19) = (   0.8750000  -0.1250000   0.8750000), wk =   0.0937500
        k(   20) = (   0.7500000   0.0000000   0.7500000), wk =   0.0468750
        k(   21) = (   0.0000000  -1.0000000   0.0000000), wk =   0.0117188
        k(   22) = (  -0.2500000   0.5000000   0.0000000), wk =   0.0937500
        k(   23) = (   0.6250000  -0.3750000   0.8750000), wk =   0.1875000
        k(   24) = (   0.5000000  -0.2500000   0.7500000), wk =   0.0937500
        k(   25) = (   0.7500000  -0.2500000   1.0000000), wk =   0.0937500
        k(   26) = (   0.6250000  -0.1250000   0.8750000), wk =   0.1875000
        k(   27) = (   0.5000000   0.0000000   0.7500000), wk =   0.0937500
        k(   28) = (  -0.2500000  -1.0000000   0.0000000), wk =   0.0468750
        k(   29) = (  -0.5000000  -1.0000000   0.0000000), wk =   0.0234375

     G cutoff =  344.1848  (   6735 G-vectors)     FFT grid: ( 27, 27, 27)
     G cutoff =  114.7283  (   1243 G-vectors)  smooth grid: ( 15, 15, 15)

